<ui version="4.0" >
 <class>RoiPrefsDlg</class>
 <widget class="QDialog" name="RoiPrefsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ROI Preferences</string>
  </property>
  <widget class="QPushButton" name="RoiOkButton" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>140</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>130</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>20</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Colors</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>ROI Line Color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>ROI Handles Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QPushButton" name="RoiLineButton" >
       <property name="text" >
        <string/>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QPushButton" name="RoiHandlesButton" >
       <property name="text" >
        <string/>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
